### ISSUE TITLE
AttributeError: 'Retrying' object has no attribute 'call'

### ISSUE THREAD

**Issue by matt-ultra** [socialwifi/jsonapi-requests#55]
trying to make a post to a drupal site and got this error.

```
import requests
from requests.structures import CaseInsensitiveDict
import json
import jsonapi_requests

url = "https://domain.com/node?"
author = "Matt S"
is_tagged = 0
if is_tagged == 0:
    tagged = 2
else:
    tagged =  1
jobTitle = "horizon bank ZIP 6"
pages = 10
imageCount = 20
fieldCount = 50
filename = "sample.pdf"
path = "/var/www/docflow/privitefiles/2022-2/11"

api = jsonapi_requests.Api.config({
    'API_ROOT': 'https://fomain.com/jsonapi/node/job_detail',
    'AUTH': ('user', 'password'),
    'VALIDATE_SSL': False,
    'TIMEOUT': 1,
})

endpoint = api.endpoint(url)


relationship = {
	"field_job_id": {
		"data": {
			"type": "node--job",
			"id":  "cf904298-2c87-44d1-b23f-75223127ed09",
			"meta": {
				"drupal_internal_target_id": 8
			}
		}
	}
}

endpoint.post(object=jsonapi_requests.JsonApiObject(
    type='node--job_detail',
    attributes={
        'title': 'JSON API TITLE',
        'path': {
        	"field_author":	"Matt",
			"field_document_type":	"PDF",
			"field_number_of_fields":	5,
			"field_number_of_images_without_a":	4,
			"field_number_of_pages":	3,
			"field_number_of_scanned_image_pa":	2,
			"field_number_of_scanned_pages":	1,
			"field_number_of_total_images":	3,
			"field_original_file_location":	"/wherever/",
			"field_original_file_name":	"My original File.pdf",
			"field_pricing_category":	"4",
			"field_remediation_cost":	"12.00",
			"field_tagged":	"2"
        }
    },
    relationships= relationship
))

```

**Comment by ben221199** [comment]
Updating `tenacity` (to 7.0.0) worked for me.

**Comment by reduzent** [comment]
I had the same Problem on Debian Bullseye with Python 3.9 (It worked before on Debian 10 with Python 3.7). I had to downgrade tenacity from 8.0.0 to 7.0.0.

**Comment by jgoclawski** [comment]
The tenacity issue is fixed in version 0.7.0.
